ports/esp32s2/common-hal/wifi/__init__.c

```diff
@@ -78,12 +78,15 @@ static void event_handler(void* arg, esp_event_base_t event_base,
 
                 radio->last_disconnect_reason = reason;
                 xEventGroupSetBits(radio->event_group_handle, WIFI_DISCONNECTED_BIT);
+                break;
             }
 
             // Cases to handle later.
             // case WIFI_EVENT_STA_AUTHMODE_CHANGE:
-            default:
+            default: {
+                ESP_EARLY_LOGW(TAG, "event %d 0x%02x", event_id, event_id);
```

Comment:
We should switch these to non-early logs. That way it won't interrupt another log line. Regular ones are queued I believe.

```suggestion
                ESP_LOGW(TAG, "event %d 0x%02x", event_id, event_id);
```

Reply:
I changed all of them in the event_handler (I assume that's what you intended), there are none are elsewhere in the file